v1/formats/nswpdc7and8.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name>NSW Premier's Debating Challenge (7 and 8)</name>
+  <short-name>NSW PDC Y7–8</short-name>
+  <version>1</version>
+  <info>
+    <region>NSW, Australia</region>
+    <level>Years 7 and 8</level>
+    <used-at>NSW Premier's Debating Challenge</used-at>
+    <description>3 vs 3, 6-minute speeches, no replies, no POIs</description>
+  </info>
+  <period-types>
+   <period-type ref="pdc.1bell">
+     <name>PDC 1 Bell</name>
+     <display>1 Bell</display>
+     <default-bgcolor>#CCAA00</default-bgcolor>
+   </period-type>
+   <period-type ref="pdc.2bell">
+     <name>PDC 2 Bells</name>
+     <display>2 Bells</display>
+     <default-bgcolor>#CC6600</default-bgcolor>
+   </period-type>
+   <period-type ref="pdc.overtime">
+     <name>PDC Overtime</name>
+     <display>***OVERTIME***</display>
+     <default-bgcolor>#CC2200</default-bgcolor>
+   </period-type>
+  </period-types>
+  <prep-time length="60:00"/>
+  <speech-types>
+    <speech-type ref="speech" length="7:00" first-period="normal">
+      <bell time="4:00" number="1" next-period="pdc.1bell"/>
+      <bell time="6:00" number="2" next-period="pdc.2bell"/>
+      <bell time="finish" number="60" next-period="pdc.overtime"/>
```

Comment:
One more thing... this makes me nervous, I don't think I had intended for this to be how this would work, and it seems to interfere with the "flash screen" bells (you can enable these in settings). I gather you want continuous bells to be standard across your competition, though?

If you're trying to enforce everyone having continuous bells, then I can tolerate this, just be aware that it'll do funky things. For example, I think the bell logic cancels all previous bells when a new bell starts, so if a user's own grace period is configured to (say) 15 seconds, in practice you'll get continuous bells until 15 seconds, and then the new overtime bell will take over, even though the 60 bells requested by the format hasn't finished. And if you have flash screen enabled... ngl, I don't actually know how it's handling this situation.

(I guess this bolsters the case that overtime bells ought to be format-specified, not user-configurable!)